app/src/main/java/br/com/mob1st/bet/features/competitions/domain/Answer.kt

```diff
@@ -0,0 +1,46 @@
+package br.com.mob1st.bet.features.competitions.domain
+
+import br.com.mob1st.bet.core.utils.objects.Duo
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ * A confrontation can have many possible answers, each of them have a different [weight] and
+ * different [odds].
+ *
+ * For example, in a football match the user can answer the winner, the score, the top scorer and m
+ * more. Each one of them is a different answer, containing different weights.
+ */
+@Serializable
+sealed interface Answer<T : Any> {
+    val weight: Int
+    val odds: Odds
```

Comment:
Chegou a ver de onde vai pegar as odds?

Reply:
Sim!!! da mesma API que a gente ta pegando os restos da info
Depois no futuro a gente pode buscar odds de mais lugares.

A única dúvida que eu to ainda é como vamos fazer pra atualizar essas odds diariamente (e tb como vamos atualizar o placar dos jogos). Vou começar a trabalhar nisso nos próximos dias